roadmap.md

```diff
@@ -20,8 +20,9 @@ The PELUX project aims to provide biannual releases. The first release was done
 in January 2018. For already released versions, see the [downloads](/downloads)
 page.
 
-### PELUX 2.0 [July 2018]
+### PELUX 3.0 [September 2018]
 Content of this release is under discussion. Tentatively the topics in focus will be:
-- Intel ARP board support
-- Qt 5.10
-- SOTA updates
+- Adaptation to Yocto "sumo" 2.5
```

Comment:
I thought "sumo" was the name of the release. What is "2.5"?

Reply:
Ok. I found it on the yocto release page, so disregard my comment, but for clarity I would prefer to write it
"Adaptions for Yocto "sumo" (2.5)"